C-Plus-Plus/ds/Sparse_Matrix_Representation.cpp

```diff
@@ -0,0 +1,105 @@
+// C++ program for Sparse Matrix Representation
+// using Array
+
+
+//A matrix is a two-dimensional data object made of m rows and n columns,
+//therefore having total m x n values. If most of the elements
+//of the matrix have 0 value, then it is called a sparse matrix.
+#include <iostream>
+using namespace std;
+class Element
+{
+public:
+    int i;
+    int j;
+    int x;
+};
```

Comment:
Please change the variables names to proper words instead of just alphabets.

Reply:
I resolve this issue please have a look @Abhijit2505

Reply:
I resolve this issue please have a look @Abhijit2505